primo/utils/override_utils.py

```diff
@@ -130,43 +134,39 @@ def return_value(self):
         return self.well_list
 
 
-def UserInput(well_df_add: pd.DataFrame, well_df_remove: pd.DataFrame):
+def user_input(well_df_add: pd.DataFrame, well_df_remove: pd.DataFrame):
```

Comment:
The standard naming convention for functions is `snake_case`.

Reply:
Thank you for the comment!